libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1493,6 +1508,10 @@ function baro:get_temperature() end
 ---@return number
 function baro:get_pressure() end
 
+-- desc
+---@return number
```

Comment:
Kind of sparse documentation, right?  Knowing what *sort* of altitude is returned would be nice to have in the documentation (AMSL would be most outsider's guess, and *I'm* guess that that guess would be wrong).

Is there facility for more information in these documentation strings?

Reply:
Yes, just add more text on the same line. There are some that are more thoroughly documented, would be good to get them all fully documented eventually.